localstack-core/localstack/http/resources/swagger/endpoints.py

```diff
@@ -7,10 +7,17 @@
 from localstack.http import Response
 
 
+def _get_service_url(request: Request) -> str:
+    # special case for ephemeral instances
+    if "sandbox.localstack.cloud" in request.host:
+        return external_service_url(protocol="https", port=443)
```

Comment:
**Thought**: This seems a bit ugly - I do wonder if there's something better we can use to use port 443.

As an idea: Could we set the port for the `external_service_url` call depending on the `request.scheme`? If it's HTTPS use 443 - if not 4566.

Reply:
If we do this, it messes up with the default `https://localhost.localstack.cloud:4566`. Ephemeral instances are the only case that comes to mind where we don't have a port but just default to 443.